app/perf_overlay.conf

```diff
@@ -1,2 +1 @@
 CONFIG_PERFORMANCE_COUNTERS=y
-CONFIG_PERFORMANCE_COUNTERS_RUN_AVERAGE=y
```

Comment:
Why do we not want to know average ? @singalsu ?

Reply:
Yes, please keep average included, it's important for power consumption estimates and such. We need to know both average and max.

Also min. would be valuable. With all min/max/average we get good view of how component uses processing capablity. Why not add it too?

Reply:
We definitely need avg, this PR just want to remove the avg macro and use one macro to cover performance measurement, avg is still there, use two macro seems redundant, one is ok.

Reply:
Please @mmaka1 also take a look.

Reply:
> We definitely need avg, this PR just want to remove the avg **macro** and use one macro to cover performance measurement, avg is still there, use two macro seems redundant, one is ok.

ah, got you  - can you reword the commit message to state we are removing a duplication.

Reply:
> > We definitely need avg, this PR just want to remove the avg **macro** and use one macro to cover performance measurement, avg is still there, use two macro seems redundant, one is ok.
> 
> ah, got you - can you reword the commit message to state we are removing a duplication.

Done